crypto/ct/ct_log.c

```diff
@@ -259,6 +259,15 @@ int CTLOG_STORE_load_file(CTLOG_STORE *store, const char *file)
     return ret;
 }
 
+int CTLOG_STORE_add_log(CTLOG_STORE *store, CTLOG *log)
```

Comment:
Probably we should call this `CTLOG_STORE_add0_log`. It is OpenSSL convention to use set0/add0 when ownership of the object is being passed.

Reply:
Done.